.github/workflows/builds.yml

```diff
@@ -648,6 +658,8 @@ jobs:
         with:
           name: prep
           path: prep
+      - name: Check if the nDPI plugin was installed
+        run: test -e /usr/local/lib/suricata/ndpi.so
```

Comment:
We should run some pcap with it like https://github.com/OISF/suricata/pull/12471/files#diff-a76cf7978f0a981f911e8d68d2351a72a268977304612226433df4fb8203b06fR194 (this is done for other plugins of yours if I am correct)

Reply:
Every S-V test that doesn't specify a custom `suricata.yaml` is run with this plugin enabled. Which is why we needed the entry in the schema.